playbooks/eucalyptus_basic_server.yml

```diff
@@ -3,7 +3,7 @@
 # This file based on `edx_sandbox.yml`
 
 - name: Configure instance(s)
-  hosts: all
+  hosts: edxapp-server
```

Comment:
Assuming here that insights is supposed to run on the same server as edx? (I just have a rudimentary knowledge about insights)